operators/src/processing/raster_scaling.rs

```diff
@@ -25,26 +26,33 @@ use tracing::{span, Level};
 #[derive(Debug, Serialize, Deserialize, Clone)]
 #[serde(rename_all = "camelCase")]
 pub struct RasterScalingParams {
-    slope_key_or_value: Option<PropertiesKeyOrValue>,
-    offset_key_or_value: Option<PropertiesKeyOrValue>,
+    slope: PropertiesKeyOrValue,
+    offset: PropertiesKeyOrValue,
     output_measurement: Option<Measurement>,
     scaling_mode: ScalingMode,
 }
 
 #[derive(Debug, Serialize, Deserialize, Clone, Copy)]
 #[serde(rename_all = "camelCase")]
 pub enum ScalingMode {
-    Scale,
-    Unscale,
+    MulSlopeAddOffset,
+    SubOffsetDivSlope,
 }
 
 #[derive(Debug, Serialize, Deserialize, Clone)]
 #[serde(rename_all = "camelCase", tag = "type")]
 enum PropertiesKeyOrValue {
+    Auto,
```

Comment:
the enum name implies that there are only two variants. Maybe name if `ScaleOffsetSelection`?

Reply:
[4c22426](https://github.com/geo-engine/geoengine/pull/757/commits/4c224264049253672d4e61c0e0a5f36dc10a54dd)